app/views/candidate_interface/content/providers.html.erb

```diff
@@ -6,7 +6,7 @@
 
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
-    <p class="govuk-body">Use <%= govuk_link_to service_name, candidate_interface_start_path %> if the courses you want to do are listed on this page.</p>
+    <p class="govuk-body">Use <%= govuk_link_to service_name, candidate_interface_create_account_or_sign_in_path %> if the courses you want to do are listed on this page.</p>
```

Comment:
I think in this instance, we’ll want to link to our start page on GOV.UK, as that has eligibility criteria etc, and should remain the start of the journey.

Reply:
@paulrobertlloyd - yep makes sense. Do we have the GOV.UK link yet? Couldn't see it in the ticket

Reply:
Yup, it’s: https://www.gov.uk/apply-for-teacher-training

Reply:
Ta!

Reply:
👍